### ISSUE TITLE
Does not work on iossimulator-arm64

### ISSUE THREAD

**Issue by rdavisau** [BranchMetrics/xamarin-branch-deep-linking-attribution#160]
### Describe the bug

Generally developers on recent-ish (M1, M2, M3) machines will build and run arm64 simulator builds (`RuntimeIdentifier: iossimulator-arm64`). Branch SDK init fails on these builds, because the internal call to `IOSNativeBranch.Branch.GetInstance(this.branchKey)` returns null. Possibly the binding is not produced with a simulator arm64 slice? Switching to `iossimulator-x64` resolves the issue, but means running an x64 build, which no one on arm64 wants to do :)

### Steps to reproduce

1. In csproj, set `<PlatformTarget>arm64</PlatformTarget><RuntimeIdentifier>iossimulator-arm64</RuntimeIdentifier>`
2. Try to init SDK
3. App crashes with NRE accessing the internal `NativeBranch` object

### Expected behavior

SDK should init successfully

### SDK Version

8

### Make and Model

iOS Simulator

### OS

Any iOS

**Comment by echo-branch** [comment]
@rdavisau 
Thanks for the heads up on this. Which version of the SDK are you using? 

I just checked the latest and the simulator framework has the two slices as expected.

```echo@SEA-MBP-echo BranchSDK.iOSBinding % file BranchSDK.xcframework/ios-arm64_x86_64-simulator/BranchSDK.framework/BranchSDK
BranchSDK.xcframework/ios-arm64_x86_64-simulator/BranchSDK.framework/BranchSDK: Mach-O universal binary with 2 architectures: [x86_64:Mach-O 64-bit dynamically linked shared library x86_64] [arm64:Mach-O 64-bit dynamically linked shared library arm64]
BranchSDK.xcframework/ios-arm64_x86_64-simulator/BranchSDK.framework/BranchSDK (for architecture x86_64):	Mach-O 64-bit dynamically linked shared library x86_64
BranchSDK.xcframework/ios-arm64_x86_64-simulator/BranchSDK.framework/BranchSDK (for architecture arm64):	Mach-O 64-bit dynamically linked shared library arm64```

**Comment by rdavisau** [comment]
@echo-branch Thanks for the quick response.

Apologies, I specified SDK 8 as a reference to dotnet 8, obviously I should have been specifying the package version, which is 9.0.1. 

Since you confirmed you already switched to xcframework, I attempted to create a minimal reproduction of the issue, and found that it does not reproduce. I spent a little time changing settings on the minimal project to match the breaking project but still could not get it to break. So, I'll close this as invalid and if I work out later there are a combination of project settings that make things not work I'll come back :)
